main.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_rds_cluster_instance" "this" {
   db_parameter_group_name         = var.db_parameter_group_name
   preferred_maintenance_window    = var.preferred_maintenance_window
   apply_immediately               = var.apply_immediately
-  monitoring_role_arn             = local.rds_enhanced_monitoring_arn
+  monitoring_role_arn             = var.create_monitoring_role ? join("", local.rds_enhanced_monitoring_arn) : var.monitoring_role_arn
```

Comment:
we should handle this check/logic on line 7 and leave this line as it currently stands (`monitoring_role_arn = local.rds_enhanced_monitoring_arn`)

Reply:
for the locals on line 7 we'll need to update:

```hcl
rds_enhanced_monitoring_arn  = var.create_monitoring_role ? join("", aws_iam_role.rds_enhanced_monitoring.*.arn) : var.monitoring_role_arn
```

Reply:
Thanks @bryantbiggs, updated, amended my commit, and rebased with the latest `master`.